dataprofiler/labelers/labeler_utils.py

```diff
@@ -184,24 +188,26 @@ def evaluate_accuracy(
                 f1_report["macro avg"][metric] = np.nan
             else:
                 f1_report["macro avg"][metric] *= (
-                    float(len(label_names)) / num_labels_with_positive_support
+                    float(len(label_names or [])) / num_labels_with_positive_support
```

Comment:
`label_names` may be `None` which would cause an error

Reply:
thoughts on putting a condition here instead of doing `or []`... add an `if not label_names: label_names = []` then do `float(len([label_names]))`